hw/ip/rstmgr/data/rstmgr_testplan.hjson

```diff
@@ -30,17 +30,183 @@
+              values.
+
+            **Checks**:
+            - Check that the zero bits in `sw_rst_ctrl_n` enabled by
+              `sw_rst_regen` cause the respective resets to become active.
+            - Check that the zero bits in `sw_rst_ctrl_n` disabled by
+              `sw_rst_regen` have no effect on resets.
+            - Check the `cpu_dump` CSR is not modified.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: reset_info
+      desc: '''Test the reporting of reset reason.
+
+            **Stimulus**:
+            - Generate the different resets recorded in `reset_info` CSR.
+            - Randomly clear `reset_info` (it is rw1c).
+
+            **Checks**:
+            - The resulting setting of `reset_info` is as expected.
+            - Each bit was set at least once.
+            - Each bit was cleared at least once.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: cpu_info
+      desc: '''Test the cpu_info recording.
+
+            The `cpu_info` CSR register(s) can capture the contents of the
+            `cpu_dump_i` input when resets happen and it is enabled.
+
+            **Stimulus**:
+            - Regularly modify the `cpu_dump_i` input.
+            - With `cpu_regwen` on, randomly set `cpu_info_ctrl.en` to control
+              whether the dump should be captured.
+            - Generate reset(s) as in `smoke` testpoint.
+
+            **Checks**:
+            - Check that `cpu_regwen` doesn't transition to 1 after reset.
+            - Verify the `cpu_info` is only captured when enabled.
+            - Verify the `cpu_info` contents at each `cpu_info_ctrl.index`
+              matches the expected value.
+
+            - Checking that `cpu_regwen` controls when `cpu_info_ctrl` can be
+              modified should be done in common csr tests.
+            '''
+      milestone: V2
+      tests: []
+    }
+    {
+      name: alert_info
+      desc: '''Test the alert_info recording.
+
+            The `alert_info` CSR register(s) can capture the contents of the
+            `alert_dump_i` input when resets happen and it is enabled.
+
+            **Stimulus**:
+            - Regularly modify the `alert_dump_i` input.
+            - With `alert_regwen` on, randomly set `alert_info_ctrl.en` to
+              control whether the dump should be captured.
+            - Generate reset(s) as in `smoke` testpoint.
+
+            **Checks**:
+            - Check that `alert_regwen` doesn't transition to 1 after reset.
+            - Verify the `alert_info` is only captured when enabled.
+            - Verify the `alert_info` contents at each `alert_info_ctrl.index`
+              matches the expected value.
+
+            - Checking that `alert_regwen` controls when `alert_info_ctrl` can
+              be modified should be done in common csr tests.
+            '''
+      milestone: V2
       tests: []
     }
   ]
```

Comment:
Would you please add the stress test like you did for clkmgr in your other PR (and likewise for pwrmgr too)? And, import the stress testplan.

Reply:
Done